src/components/FooterV2.js

```diff
@@ -68,7 +68,7 @@ const useStyles = makeStyles((theme) => ({
     color: '#fff',
```

Comment:
Looks like an incorrect shift in the footer icons:
https://www.chromatic.com/pullrequest?activeElementId=comparison-test-key-618b0e435e3b69e68a2c3429-1200&appId=6154d1bab7cd94003aee223d&number=230&view=changes

We're looking for a little more space above "social" and below the logo.